### ISSUE TITLE
Allow an increase of threads_batch to a value >32 from within the ui

### ISSUE THREAD

**Issue by spreck** [oobabooga/text-generation-webui#5292]
**Description**

I want to be able to increase the number of threads_batch to 48 in the ui.

**Additional Context**

If applicable, please provide any extra information, external links, or screenshots that could be useful.

According to these docs: (https://github.com/oobabooga/text-generation-webui/wiki/04-%E2%80%90-Model-Tab#llamacpp
llama.cpp)

I should set:

> threads: Number of threads. Recommended value: your number of physical cores.
> threads_batch: Number of threads for batch processing. Recommended value: your total number of cores (physical + virtual).

I have 24 physical cores and 48 virtual cores and I want to be able to use them all. I can enter 48 in the Model Tab threads_batch area but as soon as I click away it reverts to a maximum of 32.

**Comment by chraac** [comment]
yeah, I second this one, have a 24 cores 48 threads machine also, but in ui it can only enable 32 threads, had to change the yaml to get all 48 threads work.

**Comment by chraac** [comment]
update: increased the max_thread number in my locak fork, tested with llama.cpp, worked as expected:
https://github.com/chraac/text-generation-webui/tree/dev-max-threads

**Comment by chraac** [comment]
Created a [PR](https://github.com/oobabooga/text-generation-webui/pull/5889) about it, works well mo my machine
